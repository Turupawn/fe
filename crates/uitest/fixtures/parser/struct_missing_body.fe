
struct S

struct T<U>

struct Foo {}
