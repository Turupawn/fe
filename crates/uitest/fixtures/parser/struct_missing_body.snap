---
source: crates/uitest/tests/parser.rs
expression: diags
input_file: crates/uitest/fixtures/parser/struct_missing_body.fe
---
error[1-0001]: expected `{`, `where` or `<`
  ┌─ struct_missing_body.fe:2:9
  │
2 │ struct S
  │         ^ expected `{`, `where` or `<`

error[1-0001]: expected `{` or `where`
  ┌─ struct_missing_body.fe:4:12
  │
4 │ struct T<U>
  │            ^ expected `{` or `where`


